interview-questions.md

```diff
@@ -7,65 +7,74 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. Rails follows an MVC pattern. What does that mean?
 
-  Your answer:
+  Your answer: MVC stands for model view controller and it is a software design pattern. I know it breaks things into three components
 
-  Researched answer:
+  Researched answer: It seems that it is a design that divides the related logic into three elements which are the model view and controller. The user sees the view and uses the controller which manipulates the model which when updated changes the view which the uses then sees.
 
 
 
 2. What does it mean if an application has CRUD? What does CRUD stand for?
 
-  Your answer:
+  Your answer: CRUD stands for create, read, update, and delete which are the requirements for a full-stack application
 
-  Researched answer:
+  Researched answer: Pretty much what my answer said but these four functions are considered necessary to implement a persistent storage application.
 
 
 
 3. What is a relational database? Are there other kinds of databases?
 
-  Your answer:
+  Your answer: Only know that it will most likely have to do with relationship between two different data bases.
```

Comment:
not necessarily between two different databases. It's more about the relationship between tables within one database